packages/docusaurus-plugin-content-pages/package.json

```diff
@@ -22,6 +22,7 @@
     "@hapi/joi": "17.1.1",
     "globby": "^10.0.1",
     "loader-utils": "^1.2.3",
+    "minimatch": "^3.0.4",
```

Comment:
We are already using a deps called globby in other places, maybe it could do the job here too instead of this new lib?

Reply:
actually this is not used right?

Reply:
`minimatch` is used to convert glob pattern to regex